apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -308,13 +326,16 @@ export const genChartQuery = (
   const [startOffset, trunc] = calcChartStart(params)
   const where = genWhereStatement(table, filters)
 
+  const errorCondition = getErrorCondition(table)
+
   let joins = genCrossJoinUnnests(table)
 
   return `
 SELECT
 -- log-event-chart
   timestamp_trunc(t.timestamp, ${trunc}) as timestamp,
-  count(t.timestamp) as count
+  count(t.timestamp) as count,
+  LOGICAL_OR(${errorCondition}) as has_error
```

Comment:
Not really necessary for logical_or if you aren't working with arrays, fine to just use bare condition I think wrapped in a parentheses